src/inmanta/export.py

```diff
@@ -222,7 +222,22 @@ def _call_dep_manager(self, types: ModelDict) -> None:
             except UnknownException:
                 LOGGER.debug("Dependency manager %s caused an unknown exception", fnc)
 
-        # TODO: check for cycles
+        def cleanup(requires: Union[str, Resource, Id]) -> Id:
+            if isinstance(requires, str):
+                return Id.parse_id(requires)
```

Comment:
```suggestion
                return Id.parse_id(ResourceIdStr(requires))
```
Also requires import

Reply:
Perhaps I should document here that we know that the type can be violated here and this may raise an exception.

Reply:
The cleanup function, you mean? Because this line will not violate a type, right? The proposed change at least.

Reply:
Done